extensions/hp-extension/src/custom-viewport/setViewportZoomPan.ts

```diff
@@ -5,30 +5,67 @@
  * @param value 
  * @param viewport 
  */
-const setViewportZoomPan = (id,value,viewport) => {
-  const {zoom,pan, flipHorizontal, flipVertical} = value;
-  if( zoom && zoom!==1 ) viewport.setZoom(zoom);
-  if( pan ) viewport.setPan(pan);
-  if( flipHorizontal || flipVertical ) {
+const setViewportZoomPan = (id, value, viewport) => {
```

Comment:
Need to apply the imageCanvasPoint and initialDisplay area manually due to PR changes on CS3D